source/fundamentals/crud/read-operations/query-document.txt

```diff
@@ -443,8 +441,8 @@ Query Operators </reference/operator/query-bitwise/>` page.
 Additional Information
 ----------------------
 
-.. For information on specifying a geospatial query, see the guide on
-.. :doc:`Geospatial Data </fundamentals/crud/read-operations/geospatial>`.
+For information on specifying a geospatial query, see the guide on
+:ref:`golang-geo`.
```

Comment:
Suggestion: Uncomment and add link text:
```suggestion
For information on specifying a geospatial query, see the guide on
:ref:`Geospatial Data <golang-geo>`.
```

Reply:
back to you!